SentinelOne/connector_sentinelone_logs.json

```diff
@@ -0,0 +1,44 @@
+{
+    "description": "SentinelOne logs connector",
+    "docker_parameters": "sentinelone_log_connector",
+    "name": "SentinelOne logs connector",
+    "arguments": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "properties": {
+            "client_id": {
+                "description": "",
+                "type": "string"
+            },
+            "client_secret": {
+                "description": "",
+                "type": "string"
+            },
+            "api_key": {
+                "description": "",
+                "type": "string"
+            },
+            "management_domain": {
+                "description": "",
+                "type": "string"
+            },
```

Comment:
The module configuration already defines the APIKey and the Hostname/Management domain.
I can/must resuse them.
```suggestion
```